linux/mozillavpn.spec

```diff
@@ -28,6 +28,9 @@ BuildRequires: qt6-qtsvg-devel >= 6.0
 BuildRequires: qt6-qttools-devel >= 6.0
 BuildRequires: qt6-qtwebsockets-devel >= 6.0
 BuildRequires: qt6-qt5compat-devel >= 6.0
+BuildRequires: qt6-qtbase-mysql >= 6.0
```

Comment:
Why do we need these 3? We don't use them. Just SQLite.

Reply:
To use Qt6Sql cmake files these plugins have to be present during configuration stage, here is an example of an error that happens without these packages:

https://github.com/mozilla-mobile/mozilla-vpn-client/actions/runs/3849926939/jobs/6560059259

I haven't found a single package that would install those drivers as dependencies, so I've added them to build dependencies one by one. There might be another way around this to avoid installing them, but I don't know of any.

Reply:
I've run into the same thing when playing around with RPM builds in taskcluster. I don't know why this became a problem in Fedora 37 and onwards, but so be it :)

Reply:
One thing worth checking is whether we need to add a `Requires: qt6-qtbase-mysql` as well for Qt 6.4.x. You might want to check the logs to see if there are any warnings about the SQL driver not being found as this can affect how data is stored by QML.

Reply:
This doesn't seem to be a problem - I'm currently using the RPM build from most recent workflow run on this PR and I've only got sqlite driver installed on my system.

Reply:
Right, and also no warnings in the debug logs about any missing db drivers.